herbert_core/admin/@opt_config_manager/OptimalConfigInfo.xml

```diff
@@ -242,15 +275,15 @@
       </herbert_config>
       <hor_config>
          <mem_chunk_size>20000000</mem_chunk_size>
-         <threads>8</threads>
+         <threads>2</threads>
```

Comment:
does @hsautessella fix to logical vs physical cores mean that it's possible to run the ANVIL Linux nodes with `>2` threads?

Reply:
That fix is not merged https://github.com/pace-neutrons/Herbert/pull/102

Reply:
at the moment it Horace tests was failing on Linux as it requested 5-6 workers
no Matlab parpool or mpiexec for some reason can run this. I wander how this test would be able

Reply:
I believe, tests for cores and threads are different one, so there are range of omp tests which would use threads, whilc MPI would use both. Let it be too.